cinema/models.py

```diff
@@ -110,14 +104,10 @@
         update_fields=None,
     ):
         self.full_clean()
-        super(Ticket, self).save(
-            force_insert, force_update, using, update_fields
-        )
+        super(Ticket, self).save(force_insert, force_update, using, update_fields)
```

Comment:
Calling `self.full_clean()` in the `save` method is a good practice to ensure data integrity. Make sure that any exceptions raised by `full_clean` are handled appropriately in your application logic.